omero/developers/Modules/searchfieldnames.tsv

```diff
@@ -11,7 +11,7 @@ details.updateEvent.id	Id of the Event of this objects last modification
 details.updateEvent.time	When that Event took place
 details.permissions	Permissions in the form `rwrwrw` or `rw-`
 tag	Contents from a ``TagAnnotation``.
-annotation	Contents from any annotations, including ``TagAnnotation`` and any ``TextAnnotation`` on another ``TextAnnotation`` (a.k.a. a `description`). Non-string annotations like file annotations are not covered by this definition and are handled separately. See below.
+annotation	Contents from annotations, including ``TagAnnotation`` and any ``TextAnnotation`` on another ``TextAnnotation`` (a.k.a. a `description`). Non-string annotations like ``FileAnnotations`` are not covered by this definition and are handled separately. See below.
```

Comment:
I am not sure we should have a *literal* `FileAnnotations` as the only literal form in the code is the singular.